_overviews/scala3-migration/tooling-syntax-rewriting.md

```diff
@@ -92,13 +96,19 @@ We will be able to move it to new syntax automatically in two steps: first by us
 
 We can use the `-new-syntax -rewrite` options by adding them to the list of scalac options in our build tool.
 
+{% tabs scala-3-location_1 %}
+{% tab 'Scala 3 Only' %}
```

Comment:
Since this is the `build.sbt` file there is no need to surround this with tabs, unless you want to label it "sbt"